app/javascript/dashboard/api/ApiClient.js

```diff
@@ -10,6 +10,10 @@ class ApiClient {
   }
 
   get url() {
+    return `${this.baseUrl()}/${this.resource}`;
```

Comment:
Why was this change made?

Reply:
need the baseUrl in one of the classes that inherit API client since one of the action URLs is a slightly different pattern than whats originally passed to the constructor.